types/timepicker/index.d.ts

```diff
@@ -0,0 +1,162 @@
+// Type definitions for Timepicker Plugin for jQuery 1.13
+// Project: <https://github.com/jonthornton/jquery-timepicker>
+// Definitions by: doberkofler <https://github.com/doberkofler>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.0
+
+/// <reference types="jquery"/>
+
+interface JQueryTimepickerOptions {
+	// Override where the dropdown is appended.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/timepicker/index.d.ts;
  Line: 10;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.